### ISSUE TITLE
Trigger Assistant manually?

### ISSUE THREAD

**Issue by lurch** [thonny/thonny#1201]
The Assistant seems useful (I've only started using Thonny today for the first time).
Is there any way to manually run the assistant? It would be nice to be able to fix some of the suggestions made by the assistant, and then re-run _just_ the assistant, without having to run my python program... :confused:  (If this **is** possible, I guess it needs to be made more obvious?)

**Comment by aivarannamaa** [comment]
Nope, this is not currently possible. I need to consider it once I incorporate features for more flexible running of multi-file programs.
